plaso/parsers/apthistory.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+"""Parser for Advanced Packaging Tool (APT) History log files."""
+
+from __future__ import unicode_literals
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import logger
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class AptHistoryLogEventData(events.EventData):
```

Comment:
It looks like the "commandline" string from the log would be useful to include - can you add this as well?